bundles/EcommerceFrameworkBundle/Model/AbstractOrder.php

```diff
@@ -111,43 +110,43 @@ public function setTotalPrice($totalPrice)
     }
 
     /**
-     * Should return a float
+     * @throws UnsupportedException
      *
-     * @return void
+     * @return float
      */
     public function getTotalNetPrice()
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('getTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('getTotalNetPrice not implemented for ' . get_class($this));
     }
 
     /**
+     * @throws UnsupportedException
+     *
      * @param float $totalPrice
      */
     public function setTotalNetPrice($totalPrice)
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('setTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('setTotalNetPrice not implemented for ' . get_class($this));
     }
 
     /**
-     * Should return an array
+     * @throws UnsupportedException
      *
      * @return array
      */
     public function getTaxInfo()
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('getTaxInfo not implemented for ' . get_class($this));
+        throw new UnsupportedException('getTaxInfo not implemented for ' . get_class($this));
     }
 
     /**
+     * @throws UnsupportedException
+     *
      * @param array $taxInfo
      */
     public function setTaxInfo($taxInfo)
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('setTaxInfo not implemented for ' . get_class($this));
+        throw new UnsupportedException('setTaxInfo not implemented for ' . get_class($this));
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7553